migtests/scripts/functions.sh

```diff
@@ -30,28 +30,52 @@ print_env() {
 run_psql() {
 	db_name=$1
 	sql=$2
-	psql "postgresql://${SOURCE_DB_USER}:${SOURCE_DB_PASSWORD}@${SOURCE_DB_HOST}:${SOURCE_DB_PORT}/${db_name}" -c "${sql}"
+	psql "postgresql://postgres:secret@${SOURCE_DB_HOST}:${SOURCE_DB_PORT}/${db_name}" -c "${sql}"
+}
+
+grant_user_permission_postgresql() {
+	db_name=$1
+
+	echo "SELECT 'GRANT USAGE ON SCHEMA ' || schema_name || ' TO ${SOURCE_DB_USER};' FROM information_schema.schemata; \gexec" | psql "postgresql://postgres:secret@${SOURCE_DB_HOST}:${SOURCE_DB_PORT}/${db_name}" 
```

Comment:
Can you try using `run_psql()` instead of `echo ... | psql ...`?

Reply:
I wasn't actually able to use run_psql(). I had tried it earlier.